DependencyInjection/Configuration.php

```diff
@@ -35,7 +35,11 @@ public function getConfigTreeBuilder()
             $root = $tb->root('jms_serializer')->children();
         }
 
-        $root->booleanNode('profiler')->defaultTrue($this->debug)->end();
+        if ($this->debug) {
```

Comment:
what about `$root->booleanNode('profiler')->defaultValue($this->debug)->end();` ?

Reply:
@goetas It looks like we there is already PR for it: https://github.com/schmittjoh/JMSSerializerBundle/pull/892/files